spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -144,6 +147,44 @@ object external {
     Seq(dbEntity)
   }
 
+  // ================ RDBMS based entities ======================
+  val RDBMS_TABLE = "rdbms_table"
```

Comment:
Maybe we could have RDBMS_DATABASE as well? Given that we are tracking DB for Spark/Hive table as well as NAMESPACE for HBase.

Reply:
I tried doing this using rdbms_db, however during manual testing I would get an error from Atlas about the this was unknown entity. Have you experienced something similar before?

Reply:
Please ignore my comment. We can start with only table and add it when we need.